src/track/track.ts

```diff
@@ -72,6 +72,7 @@ export class ScrollbarTrack implements I.ScrollbarTrack {
       display: pageSize <= containerSize ? 'none' : 'block',
     });
 
-    this.thumb.update(scrollOffset, containerSize, pageSize);
+    const trackHeightOffset = (this.direction === TrackDirection.Y ? this.element.offsetHeight : this.element.offsetWidth) - containerSize;
```

Comment:
Why is this needed?

Reply:
For cases when scrollbar height/width is not equal to scrollable area size, e.g.

![dialog_layout](https://user-images.githubusercontent.com/5558549/133737708-d6919093-c56c-4171-9b50-3f4177bac7cd.png)

Reply:
Then I think the thumb size would be `trackSize / pageSize` in that case, so leaving the 3rd parameter as `pageSize` is enough (the next line). Have you tested it?